src/renderer/redux/sagas/win.ts

```diff
@@ -5,14 +5,19 @@
 // that can be found in the LICENSE file exposed on Github (readium) in the project repository.
 // ==LICENSE-END==
 
-import { SagaIterator } from "redux-saga";
-import { put, take } from "redux-saga/effects";
-
+import { i18nActions } from "readium-desktop/common/redux/actions/";
 import { winActions } from "readium-desktop/renderer/redux/actions";
+import { SagaIterator } from "redux-saga";
+import { all, put, take } from "redux-saga/effects";
 
 export function* winInitWatcher(): SagaIterator {
     while (true) {
-        yield take(winActions.initRequest.ID);
+
+        yield all({
+            win: take(winActions.initRequest.ID),
+            i18n: take(i18nActions.setLocale.ID),
+        });
+
```

Comment:
When looking at this PR #902 in isolation, it is not clear why combining `winActions.initRequest` and `i18nActions.setLocale` is so important to trigger `winActions.initSuccess`.

So, I would like to suggest closing this PR, and instead promote PR #903 as a *complete* self-explanatory solution to issues #882 and #897. The critical code change is evident in this particular diff:
https://github.com/readium/readium-desktop/pull/903/files#diff-c5e985b08bbba519929349eeff28018fR19

@panaC what do you think?

Reply:
you're right